cmd/skaffold/app/cmd/flags.go

```diff
@@ -254,15 +254,15 @@ var flagRegistry = []Flag{
 		Value:         &opts.KubeContext,
 		DefValue:      "",
 		FlagAddMethod: "StringVar",
-		DefinedOn:     []string{"build", "debug", "delete", "deploy", "dev", "run"},
+		DefinedOn:     []string{"build", "debug", "delete", "deploy", "dev", "run", "filter"},
```

Comment:
This feels weird, but we do derive the local docker daemon from the kube-context (e.g., minikube)